### PR TITLE
added linkwarden configs

### PR DESCRIPTION
**Pull request by jlssmt**
[linuxserverurl]: https://linuxserver.io
[linuxserver.io][linuxserverurl]

------------------------------

 - [x] I have read the [contributing](https://github.com/linuxserver/reverse-proxy-confs/blob/master/.github/CONTRIBUTING.md) guideline and understand that I have made the correct modifications

------------------------------

## Description
added linkwarden configs

## Benefits of this PR and context
added linkwarden configs

## Source / References
sample as base

### CODE REVIEW

**Comment by jlssmt** [comment]
no interest here?
linkwarden has 7.6k github stars

**Review by Roxedus — CHANGES_REQUESTED** [review]
This has a api that might be interesting to consume. add a block excluding the api from eternal auth.

**Comment by jlssmt** [comment]
> This has a api that might be interesting to consume. add a block excluding the api from eternal auth.

like this?

**Review by Roxedus — APPROVED** [review]

**Review by Roxedus — CHANGES_REQUESTED** [review]

File: linkwarden.subfolder.conf.sample [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+    set $upstream_proto http;
+    proxy_pass $upstream_proto://$upstream_app:$upstream_port;
+
+}
+
+location ~ (/linkwarden)?/api {
+    include /config/nginx/proxy.conf;
+    include /config/nginx/resolver.conf;
+    set $upstream_app linkwarden;
+    set $upstream_port 80;
```
[the remark this answers is not in the document]
Comment (jlssmt) [reply linkwarden.subfolder.conf.sample]: yes you're right.
changed

**Pull request merged**